gluten-core/src/test/scala/org/apache/spark/softaffinity/SoftAffinitySuite.scala

```diff
@@ -47,7 +47,7 @@ class SoftAffinitySuite extends QueryTest with SharedSparkSession with Predicate
       ).toArray
     )
 
-    val locations = SoftAffinityUtil.getFilePartitionLocations(partition)
+    val locations = SoftAffinityUtil.getFilePartitionLocations(partition, partition.files)
```

Comment:
The modification here is because the partition file of iceberg cannot be obtained directly through `icebergPartition.files`.

https://github.com/liujiayi771/gluten/blob/df0d79568da5561ba5b45a425903f21922558cc3/backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/IteratorHandler.scala#L127